cmd/modern/root/install/mssql-base_test.go

```diff
@@ -0,0 +1,32 @@
+package install
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDbNameIfExists(t *testing.T) {
+
+	type test struct {
+		input          string
+		expectedOutput string
+	}
+
+	tests := []test{
+		// Positive Testcases
+		{"https://example.com,myDbName", "myDbName"},
+		{"https://example.com,myDb Name", "myDb Name"},
+		{"https://example.com,myDb Na,me", "myDb Na,me"},
+		{"https://example.com,[myDb Na,me]", "[myDb Na,me]]"},
+		{"https://example.com,[myDb Na'me]", "[myDb Na''me]]"},
```

Comment:
>'' [](http://example.com/codeflow?start=48&length=2)

The '``should only be doubled if this string is also being injected into the EXEC  dynamic SQL parameter.

Reply:
This is indeed the case in our code.